at_response.c

```diff
@@ -1229,6 +1229,8 @@ static int at_response_cmgr (struct pvt* pvt, const char * str, size_t len)
 				return 0;
 			}
 
+			ast_log(LOG_NOTICE, "[%s] Incoming message parsed '%s'\n", PVT_ID(pvt), str);
+
 			ast_debug (1, "[%s] Successfully read SMS message\n", PVT_ID(pvt));
```

Comment:
O:-) @wdoekes keeps an eye on the code.

Thank you for this exchange that helped me find a better solution: use ${CMGR} in extensions.conf and send content to a file, by e-mail, to an external process, ...

Can cancel / close this pull requests ?